src/components/Form.js

```diff
@@ -118,7 +133,12 @@ export default class Form extends Component {
   };
 
   render = () => {
-    return <div ref={element => this.element = element} />;
+    return (
+      <div>
+        <Loader isLoading={this.state.isLoading}/>
```

Comment:
I don't think we need to store isLoading in state. We should be able to get it directly from props since it is a part of the reducers already. See isActive in react-formio modules. We can just use that after mapStateToProps.